src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -51,6 +51,7 @@ object Boot extends App with LazyLogging {
     val monitorConfig = config.as[MonitorConfig]("monitor")
     val samConfig = config.as[SamConfig]("sam")
     val autoFreezeConfig = config.as[AutoFreezeConfig]("autoFreeze")
+    val contentSecurityPolicy = config.as[Option[String]]("jupyterContentSecurityPolicy.contentSecurityPolicy").getOrElse("default-src: 'self'")
```

Comment:
See comment above, would be nice to access this like `"jupyterConfig.contentSecurityPolicy"`.